pkg/resource/unstructured/client.go

```diff
@@ -54,7 +54,7 @@ type WrapperClient struct {
 // Get retrieves an obj for the given object key from the Kubernetes Cluster.
 // obj must be a struct pointer so that obj can be updated with the response
 // returned by the Server.
-func (c *WrapperClient) Get(ctx context.Context, key client.ObjectKey, obj client.Object) error {
+func (c *WrapperClient) Get(ctx context.Context, key client.ObjectKey, obj client.Object, opts ...client.GetOption) error {
```

Comment:
Is it intentional that you are not using opts?

Reply:
No, good catch.